config/tools-automated.json

```diff
@@ -75,21 +73,6 @@
   "Code Generators": {
     "description": "The following is a list of tools that generate code from an AsyncAPI document; not the other way around.",
     "toolsList": [
-      {
-        "title": "Golang AsyncAPI Code Generator",
-        "description": "Generate Go client and server boilerplate from AsyncAPI specifications. Can be called from `go generate` without requirements.\n",
-        "links": {
-          "repoUrl": "https://github.com/lerenn/asyncapi-codegen"
-        },
-        "filters": {
-          "language": "golang",
-          "categories": [
-            "code-generator"
-          ],
-          "hasCommercial": false,
-          "isAsyncAPIOwner": false
-        }
-      },
```

Comment:
@derberg Many of the tools like this are no more recognizable from GitHub code search.  I don't know why but for every GET call of API, there are only 14 results.

Reply:
for now just update in the way to make sure it is not removed and we can merge I guess